api/config.go

```diff
@@ -27,6 +29,8 @@ type Config struct {
 	WebsocketRateLimit int `yaml:"websocketRateLimit"`
 	// ListenerLimit is the maximum number of listeners.
 	ListenerLimit int `yaml:"listenerLimit"`
+	// ReadyDuration is the duration to wait for the server to be ready.
+	ReadyDuration time.Duration `yaml:"readyDuration"`
```

Comment:
where is this value set? I also don't see a default value

Reply:
0 is default value, which means no delay